go_expvar/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 1.0.2 / Unreleased
 ==================
 
+* [BUGFIX] Fix path tag appended to all metrics matching path regex
```

Comment:
could you put this line in a new `1.0.3` section? `1.0.2` is what was shipped with `5.14.0`